src/components/main-view/main-view.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import axios from 'axios';
+
+import { MovieCard } from '../movie-card/movie-card';
+import { MovieView } from '../movie-view/movie-view';
+
+export class MainView extends React.Component {
+	constructor() {
+		//Call superclass constructor so React can initialize it
+		super();
+
+		//Initialize state to an empty object
+		this.state = {
+			movies: null,
+			selectedMovie: null,
+		};
+	}
+
+	componentDidMount() {
+		axios
+			.get('https://bchanmyflix.herokuapp.com/movies')
+			.then((response) => {
+				this.setState({
+					movies: response.data,
+				});
+			})
+			.catch(function (errror) {
```

Comment:
typo error should be `error`

Reply:
Thank you for catching this! I've corrected this issue.